cmake/FindBlosc.cmake

```diff
@@ -0,0 +1,12 @@
+find_path(Blosc_INCLUDE_DIR blosc.h)
+
+find_library(Blosc_LIBRARY NAMES blosc)
+
+if (Blosc_INCLUDE_DIR AND Blosc_LIBRARY)
+    set(Blosc_FOUND TRUE)
+    set(Blosc_INCLUDE_DIRS ${Blosc_INCLUDE_DIR})
+    set(Blosc_LIBRARIES ${Blosc_LIBRARY})
+    message(STATUS "Found Blosc library: ${Blosc_LIBRARIES}")
```

Comment:
Ditto for `Blosc_FOUND`, `Blosc_INCLUDE_DIRS`, `Blosc_LIBRARIES`, `Blosc_INCLUDE_DIR` and `Blosc_LIBRARY`.

Reply:
Ok.  That's inside CMake config files, so no problem with me.